Macos C++ test is failing on cpp-benchmark

See: https://source.cloud.google.com/results/invocations/9287e8f1-ccc5-40f1-82ce-92abd791d6d8/targets/protobuf%2Fgithub%2Fmaster%2Fmacos%2Fcpp%2Fpresubmit/log

```

++ cd benchmarks
++ make cpp-benchmark
g++ -std=c++11 -DHAVE_CONFIG_H -I. -I..  -I../src -I./cpp -I../third_party/benchmark/include    -Wall -Wwrite-strings -Woverloaded-virtual -Wno-sign-compare -fPIC -MT cpp/benchmark-cpp_benchmark.o -MD -MP -MF cpp/.deps/benchmark-cpp_benchmark.Tpo -c -o cpp/benchmark-cpp_benchmark.o `test -f 'cpp/cpp_benchmark.cc' || echo './'`cpp/cpp_benchmark.cc
In file included from cpp/cpp_benchmark.cc:33:
../third_party/benchmark/include/benchmark/benchmark_api.h:19:5: warning: the benchmark_api.h header has been deprecated and will be removed, please include benchmark.h instead [-W#warnings]
#   warning the benchmark_api.h header has been deprecated and will be removed, please include benchmark.h instead
    ^
cpp/cpp_benchmark.cc:34:10: fatal error: 'benchmarks.pb.h' file not found
#include "benchmarks.pb.h"
         ^~~~~~~~~~~~~~~~~
1 warning and 1 error generated.
make: *** [cpp/benchmark-cpp_benchmark.o] Error 1
```

--- a/benchmarks/Makefile.am
+++ b/benchmarks/Makefile.am
@@ -109,6 +109,7 @@ cpp_benchmark_CPPFLAGS = -I$(top_srcdir)/src -I$(srcdir)/cpp -I$(top_srcdir)/thi
 # so a direct "make test_cpp" could fail if parallel enough.
 # See: https://www.gnu.org/software/automake/manual/html_node/Built-Sources-Example.html#Recording-Dependencies-manually
 cpp/cpp_benchmark-cpp_benchmark.$(OBJEXT): $(benchmarks_protoc_outputs) $(benchmarks_protoc_outputs_proto2) $(benchmarks_protoc_outputs_header) $(benchmarks_protoc_outputs_proto2_header) $(top_srcdir)/src/libprotobuf.la $(top_srcdir)/third_party/benchmark/src/libbenchmark.a
+cpp/benchmark-cpp_benchmark.$(OBJEXT): $(benchmarks_protoc_outputs) $(benchmarks_protoc_outputs_proto2) $(benchmarks_protoc_outputs_header) $(benchmarks_protoc_outputs_proto2_header) $(top_srcdir)/src/libprotobuf.la $(top_srcdir)/third_party/benchmark/src/libbenchmark.a
 nodist_cpp_benchmark_SOURCES =                                             \
 	$(benchmarks_protoc_outputs)                                             \
 	$(benchmarks_protoc_outputs_proto2)                                      \

--- a/conformance/Makefile.am
+++ b/conformance/Makefile.am
@@ -279,8 +279,6 @@ $(protoc_outputs): protoc_middleman
 
 $(other_language_protoc_outputs): protoc_middleman
 
-BUILT_SOURCES = $(protoc_outputs) $(other_language_protoc_outputs)
-
 CLEANFILES = $(protoc_outputs) protoc_middleman javac_middleman conformance-java javac_middleman_lite conformance-java-lite conformance-csharp conformance-php conformance-php-c $(other_language_protoc_outputs)
 
 MAINTAINERCLEANFILES =   \

